djangocms_versioning/admin.py

```diff
@@ -1420,7 +1418,8 @@ def changelist_view(self, request, extra_context=None):
             # Check if custom breadcrumb template defined, otherwise
             # fallback on default
             breadcrumb_templates = [
-                f"admin/djangocms_versioning/{breadcrumb_opts.app_label}/{breadcrumb_opts.model_name}/versioning_breadcrumbs.html",
+                f"admin/djangocms_versioning/{breadcrumb_opts.app_label}/"
```

Comment:
Is this splitting intentional to make the lint Gods happy?

Reply:
Also, I had to add some spaces that now apparently reduce coverage?!?

Reply:
Coverage tools are stupid. Maybe we could increase line-length or explicity increase the line length of this line to bypass this linting error. What do you think?